ee/kafka_client/client.py

```diff
@@ -25,12 +25,33 @@ def flush(self):
         return
 
 
+class TestKafkaConsumer:
+    def __init__(self, topic="test", max=0, **kwargs):
+        self.max = max
+        self.n = 0
+        self.topic = topic
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        if self.n <= self.max:
+            self.n += 1
+            return f"message {self.n} from {self.topic} topic"
+        else:
+            raise StopIteration
+
+    def seek_to_beginning(self):
+        return
+
+    def seek_to_end(self):
+        return
+
+
 class _KafkaProducer:
-    def __init__(self):
-        if TEST:
+    def __init__(self, test=TEST):
+        if test:
             self.producer = TestKafkaProducer()
-        elif IS_HEROKU:
```

Comment:
will we not need this anymore at all?

Reply:
nope! We don't run anything on Heroku w/ kafka any more